examples/AWSDriverExample/src/main/java/software/amazon/DatasourceExample.java

```diff
@@ -30,18 +30,26 @@ public class DatasourceExample {
   public static void main(String[] args) throws SQLException {
     AwsWrapperDataSource ds = new AwsWrapperDataSource();
 
-    // Configure the property names for the underlying driver-specific data source:
     ds.setJdbcProtocol("jdbc:postgresql:");
 
     // Specify the driver-specific data source:
     ds.setTargetDataSourceClassName("org.postgresql.ds.PGSimpleDataSource");
 
-    // Configure the driver-specific data source:
-    Properties targetDataSourceProps = new Properties();
-    targetDataSourceProps.setProperty("serverName", "db-identifier.cluster-XYZ.us-east-2.rds.amazonaws.com");
-    targetDataSourceProps.setProperty("database", "employees");
-    targetDataSourceProps.setProperty("serverPort", "5432");
-    ds.setTargetDataSourceProperties(targetDataSourceProps);
+    // Configure basic data source information:
+    ds.setServerName("db-identifier.cluster-XYZ.us-east-2.rds.amazonaws.com");
```

Comment:
setting the server name and port via properties won't work, or is it just an alternative?

Reply:
It's just an alternative. We have the methods to set those values in the data source, but setting them via properties will still work.